### PR TITLE
Added user signups by month view back to site-stats

### PR DESCRIPTION
**Pull request by Fossil01**
Ported from nntmux

### CODE REVIEW

File: nzedb/Users.php [hunk 1]
```diff
@@ -1210,6 +1210,22 @@ public function getTopGrabbers()
 			LIMIT 10"
 		);
 	}
+	
+        /**
```
Comment (kevinlekiller) [inline nzedb/Users.php]: Indentation issues.

File: www/themes_shared/templates/admin/site-stats.tpl [hunk 2]
```diff
@@ -12,6 +12,23 @@
 		</tr>
 	{/foreach}
 </table>
+<br /><br />
+<h2>Signups</h2>
+<table style="width:100%;margin-top:10px;" class="data highlight Sortable">
+        <tr>
```
Comment (kevinlekiller) [inline www/themes_shared/templates/admin/site-stats.tpl]: Indentation issues.

File: www/admin/site-stats.php [hunk 3]
```diff
@@ -29,6 +29,9 @@
 $recent = $releases->getRecentlyAdded();
 $page->smarty->assign('recent', $recent);
 
+$usersbymonth = $page->users->getUsersByMonth();
+$page->smarty->assign('usersbymonth', $usersbymonth);
```
Comment (kevinlekiller) [inline www/admin/site-stats.php]: Possibly combine these 2 lines?
$page->smarty->assign('usersbymonth', $page->users->getUsersByMonth());

Comment (Fossil01) [inline www/admin/site-stats.php]: Used the same syntax as the lines above. For ex:

```php
$recent = $releases->getRecentlyAdded();
$page->smarty->assign('recent', $recent);
```

Comment (kevinlekiller) [inline www/admin/site-stats.php]: Alright, I usually combine those, didn't look at the rest of the file.

**Comment by Fossil01** [comment]
Fixed identation etc.

**Comment by niel** [comment]
Merged with commit 12e849

**Pull request merged**